src/components/Table/Table.js

```diff
@@ -172,13 +184,24 @@ const Table = ({ columns, data, view, actions, options }) => {
                   />
                 </TableHeader>
               ) : null}
-              {columns.map(column => (
-                <TableHeader
-                  key={`column-${column.id}`}
-                  style={filterBarActive === true ? filterBarActiveStyle : {}}>
-                  <span className="bx--table-header-label">{column.name}</span>
-                </TableHeader>
-              ))}
+              {columns.map(column => {
+                const hasSort = view.table && view.table.sort && view.table.sort.columnId === column.id;
+                return (
+                  <TableHeader
+                    key={`column-${column.id}`}
+                    style={filterBarActive === true ? filterBarActiveStyle : {}}
+                    isSortable={column.isSortable}
+                    isSortHeader={hasSort}
+                    onClick={() => {
+                      if (column.isSortable && actions.table.onChangeSort) {
+                        actions.table.onChangeSort(column.id);
+                      }
+                    }}
+                    sortDirection={hasSort ? view.table.sort.direction : 'NONE'}>
+                    <span className="bx--table-header-label">{column.name}</span>
```

Comment:
We can just use `{column.name}` without the wrapper `span` here, [`<TableHeader/>` renders children in a span](https://github.com/IBM/carbon-components-react/blob/master/src/components/DataTable/TableHeader.js#L79)

Reply:
my bad, I missed that when merging.  I've pushed a change onto the PR.